k6/vu.go

```diff
@@ -14,16 +14,19 @@ import (
 	k6lib "go.k6.io/k6/lib"
 	k6metrics "go.k6.io/k6/metrics"
 	"gopkg.in/guregu/null.v3"
-
-	"github.com/grafana/xk6-browser/common"
 )
 
-type mockVU struct {
+// VU is a k6 VU instance.
+type VU struct {
 	*k6modulestest.VU
-	loop *k6eventloop.EventLoop
+	Loop *k6eventloop.EventLoop
 }
 
-func newMockVU(tb testing.TB) *mockVU {
+// V is a convenient method for converting any value to a goja value.
+func (v *VU) V(i interface{}) goja.Value { return v.Runtime().ToValue(i) }
+
+// NewMockVU returns a mock VU.
+func NewMockVU(tb testing.TB) *VU {
```

Comment:
Since it's creating a `VU` this function should really be called `NewVU`. I think the struct should be called `MockVU` still.

Reply:
For the record, explained [here](https://github.com/grafana/xk6-browser/pull/315#discussion_r876230016).